bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java

```diff
@@ -706,20 +706,35 @@ public void initialize() {
                     "@text/offline.conf-error-no-ip-address");
         } else {
             if (hueBridge == null) {
-                if (HueBridgeConfig.HTTPS.equals(hueBridgeConfig.protocol)) {
-                    // register trustmanager service
-                    HueTlsTrustManagerProvider tlsTrustManagerProvider = new HueTlsTrustManagerProvider(
-                            ip + ":" + hueBridgeConfig.getPort(), hueBridgeConfig.useSelfSignedCertificate);
-                    serviceRegistration = FrameworkUtil.getBundle(getClass()).getBundleContext()
-                            .registerService(TlsTrustManagerProvider.class.getName(), tlsTrustManagerProvider, null);
-                }
-
                 hueBridge = new HueBridge(httpClient, ip, hueBridgeConfig.getPort(), hueBridgeConfig.protocol,
                         scheduler);
 
                 updateStatus(ThingStatus.UNKNOWN);
+
+                if (HueBridgeConfig.HTTPS.equals(hueBridgeConfig.protocol)) {
+                    scheduler.submit(() -> {
+                        // register trustmanager service
+                        HueTlsTrustManagerProvider tlsTrustManagerProvider = new HueTlsTrustManagerProvider(
+                                ip + ":" + hueBridgeConfig.getPort(), hueBridgeConfig.useSelfSignedCertificate);
+
+                        // Check before registering that the PEM certificate can be downloaded
+                        if (tlsTrustManagerProvider.getPEMTrustManager() == null) {
```

Comment:
Just a heads up, will analyze and create issue later: After upgrading to RC1 I kept an eye on my things while booting, and I ended up in this situation. So I will need to analyze exactly how it happened - but it did, and as a result, no further attempts were made to get online and I had to manually restart the binding - after which everything worked fine.